src/test/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentControllerTest.java

```diff
@@ -143,6 +153,12 @@ void addsCoverSheetIfRequired() throws Exception {
     void doesNotAddCoverSheetIfNotRequired() throws Exception {
         when(s3Service.retrieveUnsignedDocument(anyString())).thenReturn(unsignedDocument);
         when(loggingUtils.getLogger()).thenReturn(logger);
+        //
+        // ERIC headers for auth-auth
+        //
+        lenient().doReturn(ERIC_IDENTITY_HEADER_VALUE).when(request).getHeader(ERIC_IDENTITY_HEADER_NAME);
+        lenient().doReturn(ERIC_IDENTITY_HEADER_VALUE).when(request).getHeader(ERIC_IDENTITY_TYPE_HEADER_NAME);
+        lenient().doReturn(ERIC_AUTHORIZED_KEY_ROLES_VALUE).when(request).getHeader(ERIC_AUTHORIZED_KEY_ROLES );
```

Comment:
This appears to duplicate the lines 135-137? It could be extracted into a method to reduce/prevent code duplication